python/across_api/base/footprint.py

```diff
@@ -0,0 +1,100 @@
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+
+import numpy as np
+
+
+def ra_dec_to_uvec(ra, dec):
+    phi = np.deg2rad(90 - dec)
+    theta = np.deg2rad(ra)
+    x = np.cos(theta) * np.sin(phi)
+    y = np.sin(theta) * np.sin(phi)
+    z = np.cos(phi)
+    return x, y, z
+
+
+def uvec_to_ra_dec(x, y, z):
+    r = np.sqrt(x**2 + y**2 + z**2)
+    x /= r
+    y /= r
+    z /= r
+    theta = np.arctan2(y, x)
+    phi = np.arccos(z)
+    dec = 90 - np.rad2deg(phi)
+    if theta < 0:
+        ra = 360 + np.rad2deg(theta)
+    else:
+        ra = np.rad2deg(theta)
+    return ra, dec
```

Comment:
Are these sorts of functions necessary if we're storing coordinates as `SkyCoord`? You can just do use the `cartesian` attribute of a coordinate `SkyCoord` to get a unit vector representation, and convert back also.